lib/connection.c

```diff
@@ -542,7 +556,9 @@ decorrelate_jitter_next_sleep(DECORRELATE_JITTER_BACKOFF *djb, uint32 sleep) {
     {
       sleep = 4;
     }
-    return ((uint32)(sleep/2) + (uint32) (rand() % (sleep/2)));
+    // (sleep/2) + (random from 0 to sleep) = random from sleep/2 to sleep + sleep/2
+    // = sleep +-50%
+    return ((uint32)(sleep/2) + (uint32) (rand() % (sleep + 1)));
```

Comment:
nit: jitter should be added separately to make it clear or change method name to reflect this is total next sleep value.

Reply:
changed function name to get_next_sleep_with_jitter